en/docs/choreo-concepts/deployment-tracks.md

```diff
@@ -0,0 +1,51 @@
+# Deployment Tracks
+
+Deployment Tracks within Choreo are systematically structured pathways for deploying your software components. Each deployment track is Choreo's version of a CI/CD pipeline but enriched with additional capabilities aimed at simplifying developers' deployment management. These tracks ensure that your components, regardless of their origin as source code or prebuilt images, reach their intended destinations seamlessly without encountering obstacles.
+
+The primary aim of Deployment Tracks is simplification. They establish an organized and structured approach that minimizes the chances of errors and challenges that are typically associated with deployment workflows.
+
+## The significance of Deployment Tracks
+
+Deployment Tracks offer practical solutions to enhance the API consumer experience by addressing two critical challenges:
+
+- **Streamlined Deployment**: Deployment Tracks serve as well-designed routes for your software components, enhancing the organization and reliability of the deployment process, similar to a well-structured express route.
+
+- **Efficient API Versioning**: Especially beneficial for managed APIs, Deployment Tracks provide a straightforward method for creating API versions that seamlessly interact with previous iterations. This simplified version management benefits both API creators and consumers alike.
+
+## Streamlined Deployments
+
+For Streamlined Deployments, we dissect two integral approaches that leverage Deployment Tracks: the comprehensive CI/CD integration and the focused CD-Only strategy.
+
+### CI/CD with Deployment Tracks
+
+A deployment track is linked to a particular branch within a GitHub repository. This connection is useful for handling deployments to various environments. On Choreo's Deploy page, you can easily visualize the deployments to specific environments associated with your selected deployment track. Moreover, the deployment track has a functionality that initiates automatic deployments for the linked branch. When activated, merging a pull request (PR) triggers a deployment to the development environment.
+
+![Deployment Tacks - Source Repo](../assets/img/choreo-concepts/deployment-tracks-source-repo.png){.cInlineImage-half}
```

Comment:
We go with sentence style for ALT tags right?

Need to fix the typo too `Tacks` - - - > `tracks`